apps/site/test/site_web/controllers/schedule/line_test.exs

```diff
@@ -199,7 +199,7 @@ defmodule SiteWeb.ScheduleController.LineTest do
       added_dates: [],
       added_dates_notes: %{},
       description: "Weekday schedule",
-      end_date: ~D[2020-03-13],
+      end_date: ~D[2020-12-13],
```

Comment:
It'll be easier to update this in the future if you extract it to a `@modulevar`, possibly combined with `Date.add`.